Wangscape/noise/RasterBase.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include <vector>
+#include <noise/noise.h>
+#include <SFML/Graphics.hpp>
+#include <functional>
+
+namespace noise
+{
+
+class RasterBase
+{
+public:
+    typedef module::Module NoiseModule;
+    typedef double Real;
+    typedef sf::Rect<Real> Bounds;
+
+    RasterBase() = default;
```

Comment:
... Actually it is needed, because there are constructors in derived classes that use it. Reinstated.